src/client/src/components/JobApplication.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import axios from "axios";
+import "../css/jobApplication.css";
+
+
```

Comment:
Can you add a comment on what this thing does ? (i know that it is a api call to fetch the jobs but would be nice if you can add one, so that its clear for anyone who might look at this code later on)